packages/babel-traverse/src/path/family.ts

```diff
@@ -4,6 +4,34 @@ import type TraversalContext from "../context";
 import NodePath from "./index";
 import * as t from "@babel/types";
 
+const NORMAL_COMPLETION = 0;
+const BREAK_COMPLETION = 1;
+
+type Completion = {
+  path: NodePath;
+  type: 0 | 1;
```

Comment:
Should this be NORMAL_COMPLETION | BREAK_COMPLETION?

Also, any reason for not using an enum?

Reply:
I wish we could use const enum here, one of few TypeScript features Babel does not support. 😔

Reply:
We could make it
```suggestion
  type: typeof NORMAL_COMPLETION | typeof BREAK_COMPLETION;
```
?

(I don't have a preference)

Reply:
The `typeof` operator does not account for the value of the const binding, so `typeof NORMAL_COMPLETION` is `number`.

Reply:
Ok technically we could add `as const` to the declarations, but let's keep `0 | 1` at this point :joy: